utils/create_srg_export.py

```diff
@@ -249,6 +249,11 @@ def main() -> None:
                            used_rules)
         print(f"File written to {full_output}")
 
+    # export CSV to HTML table
+    import pandas as pd
```

Comment:
Do we want to have a switch for this?

Reply:
Yes, we should definitely have an option for it. Also the script is in very early stages, it was just to have a proof of concept

Reply:
updated